images/kube-proxy/v1.15.3/Dockerfile

```diff
@@ -1,7 +1,36 @@
+# https://github.com/kubernetes/kubernetes/blob/v1.18.5/build/debian-iptables/Dockerfile
```

Comment:
Does it not work to use https://github.com/kubernetes/kubernetes/blob/v1.15.3/build/debian-iptables/Dockerfile on buster?

Reply:
it does not